geode-core/src/test/java/org/apache/geode/internal/offheap/DataTypeJUnitTest.java

```diff
@@ -902,4 +904,28 @@ public void testUnknownHeaderType() throws IOException {
     String type = DataType.getDataType(bytes);
     assertThat(type).isEqualTo("Unknown header byte: " + unknownType);
   }
+
+  public static class DummyIntDataSerializableFixedID implements DataSerializableFixedID {
+    public DummyIntDataSerializableFixedID() {}
+
+    @Override
+    public int getDSFID() {
+      return 66000;
```

Comment:
Per my point above, could we refactor this `66000` into some variable or field so that it can be used throughout the tests without copy paste?

Reply:
Dan orally agreed to add the new test.